rhizome/inference_endpoint/bin/download-lb-cert

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -ueo pipefail
+
+timestamp=$(date +%s)
+new_ssl_folder="/ie/workdir/ssl_$timestamp"
+rm -rf $new_ssl_folder
+rm -rf /ie/workdir/ssl_latest
+mkdir $new_ssl_folder
+
+if [ "$1" = "self-signed" ]; then
+    openssl req -x509 -out $new_ssl_folder/ubi_cert.pem -keyout $new_ssl_folder/ubi_key.pem -newkey rsa:2048 -nodes -sha256 -days 7 -subj '/CN=localhost'
```

Comment:
This is fine, though, don't we have a bunch of self-signed certificate generation code in Ruby? Is there any benefit there to having our self-signed development regimes be systematic? Not a rhetorical question question, I haven't re-surveyed things; this may be a consistency without value (and may complicate things)

secondly, you could speed this up a by getting off RSA and using elliptic curves.  https://wiki.mozilla.org/Security/Server_Side_TLS 

Perplexity sez:

```bash
openssl req -x509 -out $new_ssl_folder/ubi_cert.pem -keyout $new_ssl_folder/ubi_key.pem -newkey ec -pkeyopt ec_paramgen_curve:P-256 -nodes -sha256 -days 7 -subj '/CN=localhost'
```

Reply:
we do have some code on the control plane side to create certificates for Postgres. This one-liner is purely to fill a gap in development, as the Load Balancer doesn't produce any certificate to download in dev environments (when no DNS server is available).

I changed it to use elliptic curves.

Reply:
yeah, I'm fine with the one liner, more wondering if there's any development ease by, say, having a shared trust root even for development.  We can worry about it later, I don't think we have any established pattern for this (and maybe we never will)